tests/test_modeling_longformer.py

```diff
@@ -352,9 +423,9 @@ def test_inference_masked_lm(self):
 
         loss, prediction_scores = model(input_ids, labels=input_ids)
 
-        expected_loss = torch.tensor(0.0620, device=torch_device)
-        expected_prediction_scores_sum = torch.tensor(-6.1599e08, device=torch_device)
-        expected_prediction_scores_mean = torch.tensor(-3.0622, device=torch_device)
+        expected_loss = torch.tensor(0.0074, device=torch_device)
```

Comment:
Previously the model calculated the wrong `attention_mask` when no `attention_mask` was given -> update the values here.